{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -236,10 +236,13 @@
 # https://django-compressor.readthedocs.io/en/latest/settings/#django.conf.settings.COMPRESS_ENABLED
 COMPRESS_ENABLED = env.bool("COMPRESS_ENABLED", default=True)
 # https://django-compressor.readthedocs.io/en/latest/settings/#django.conf.settings.COMPRESS_STORAGE
-{%- if cookiecutter.cloud_provider == 'AWS' %}
+{%- if cookiecutter.cloud_provider == 'AWS' and cookiecutter.use_whitenoise == 'y' %}
 COMPRESS_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
-{%- elif cookiecutter.cloud_provider == 'GCP' %}
+{%- elif cookiecutter.cloud_provider == 'GCP' and cookiecutter.use_whitenoise == 'y' %}
 COMPRESS_STORAGE = "storages.backends.gcloud.GoogleCloudStorage"
+{%- elif cookiecutter.cloud_provider == 'AWS' or cookiecutter.cloud_provider == 'GCP' %}
+COMPRESS_STORAGE = STATICFILES_STORAGE
```

Comment:
This is the real fix. I think we need to set this value when: 

- NOT using Whitenoise 
AND 
- Cloud provider is either AWS or GCP.

Reply:
We don't need to cover Whitenoise + no cloud provider as this tis not supported:

https://github.com/pydanny/cookiecutter-django/blob/ebd43bbd3c29ff29cdf2617e661b857b89eb520c/hooks/pre_gen_project.py#L63-L70